open-x-htb.js

```diff
@@ -128,6 +129,31 @@ function OpenXHtb(configs) {
       auidString = auidString.slice(0, -1);
       ausString = ausString.slice(0, -1);
 
+      // process tdid query param
+      var identityData = returnParcels[0].identityData;
```

Comment:
In your documentation https://knowledgebase.indexexchange.com/display/PilotsPro/How+to+Use+adsrvr.org , it's a bit vague where the identityData object would be -
> Within the returnParcels input/parameter, you can find all identity data available in the configuration under the identityData object.

Could you advice on which returnParcels 'input/parameter' it is talking about? - Currently, I'm assuming each returnParcel would have an identityData property. And if there are several returnParcel, I assume the identityData property would be the same for all of them.